learn/execution/abci.md

```diff
@@ -0,0 +1,29 @@
+# ABCI-Compatible Execution Layers in Rollkit
+
+Rollkit is designed to be modular and flexible, allowing different execution layers to be plugged in via the ABCI (Application Blockchain Interface) protocol. This means you can use a variety of ABCI-compatible applications as the execution environment for your rollup.
```

Comment:
we have a general purpose execution inteface, we should touch on the general purpose interface and then mention abci and reth as two options we support by default